config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,9 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
+]
+
+# What we had with 6.0
+#Rails.application.config.filter_parameters += [:password]
```

Comment:
👍 I think this should be fine.

You can clean up this comment.

```suggestion
Rails.application.config.filter_parameters += [
  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
]
```